postgres/init/22-portal-content.sql

```diff
@@ -161,6 +196,16 @@ ac1cf001-7fbc-1f2f-817f-bce05744000b	en
 ac1cf001-7fbc-1f2f-817f-bce0573f0009	en
 5cf74ef8-e0b7-4984-a872-474828beb5d4	en
 5cf74ef8-e0b7-4984-a872-474828beb5d4	de
+5cf74ef8-e0b7-4984-a872-474828beb5d5	en
+5cf74ef8-e0b7-4984-a872-474828beb5d6	en
+5cf74ef8-e0b7-4984-a872-474828beb5d7	en
+5cf74ef8-e0b7-4984-a872-474828beb5d7	de
+5cf74ef8-e0b7-4984-a872-474828beb5d7	pt
```

Comment:
'pt' is not a valid language-shortname. Currently we only have 'de' and 'en'.
you need to also update 21-portal-basedata.sql:
COPY portal.languages (short_name, long_name_de, long_name_en) FROM stdin;
de	deutsch	german
en	englisch	english
\.

Reply:
true, I took the wrong table. I have validated against the country table. With the new 20file structure, a DB Table connection picture will get more and more relevant I believe

Reply:
If 'language short name' is meant to be the ISO 639-1 language code (https://de.wikipedia.org/wiki/Liste_der_ISO-639-1-Codes) - (which I strongly suggest) than I agree - we should have s separate table that contains the languages where translations of portal-resources actually do exist for (the list of language-codes (which is predefined in ISO-639-1 may have more entries).